src/SFML/Main/SFMLActivity.cpp

```diff
@@ -217,7 +217,7 @@ JNIEXPORT void ANativeActivity_onCreate(ANativeActivity* activity, void* savedSt
     if (!ANativeActivity_onCreate)
     {
         LOGE("sfml-activity: Undefined symbol ANativeActivity_onCreate");
-        exit(1);
+        std::exit(1);
```

Comment:
Makes me wonder, after [recent discussion](https://github.com/SFML/SFML/pull/2458#issuecomment-1478830510) whether we should really be using `std::exit` 🤔

Reply:
Definitely not, but at least this PR will make it easier to spot where we use it.